ViewModels/PurchaseFormViewModel.cs

```diff
@@ -1,15 +1,27 @@
 ﻿using StoresManagement.Models;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 
 namespace StoresManagement.ViewModels
 {
     public class PurchaseFormViewModel
     {
+        public PurchaseFormViewModel()
+        {
+            Branches = new HashSet<Branch>();
+            Customers = new HashSet<Customer>();
+            Products = new HashSet<Product>();
+            PurchaseItems = new HashSet<PurchaseItem>();
+            Product = new Product();
```

Comment:
I did not see practice to use `HashSet<T>` in view model in contrast to entities model.  Why not `List<T>` here?
https://stackoverflow.com/questions/6391738/what-is-the-difference-between-hashsett-and-listt

Reply:
Applied.